### ISSUE TITLE
Host public alpha testnet

### ISSUE THREAD

**Issue by nathan-at-least** [Electric-Coin-Company/zcash#530]

**Comment by defuse** [comment]
Amazingly, the `zcashd` process hasn't crashed!

I pulled and fixed daemon-manage.sh to be useful for starting/stopping it again.

It didn't start the first time I tried because the proving key was a symlink to itself which lead to:

```
zcoin@zcointest:~$ cat testnet-data-dirs/0/stderr.log 


************************
EXCEPTION: St13runtime_error       
Could not open proving key file: /home/zcoin/.zcash-params/zc-testnet-public-alpha-proving.key
Please refer to user documentation for installing this file.       
bitcoin in AppInit()  
```

And:

```
zcoin@zcointest:~$ ./zcash/zcutil/fetch-params.sh 
zcash - fetch-params.sh

Updating testnet3 symlinks to regtest parameters.
Verifying parameter file integrity via sha256sum...
sha256sum: regtest/zc-testnet-public-alpha-proving.key: Too many levels of symbolic links
regtest/zc-testnet-public-alpha-proving.key: FAILED open or read
sha256sum: testnet3/zc-testnet-public-alpha-proving.key: Too many levels of symbolic links
testnet3/zc-testnet-public-alpha-proving.key: FAILED open or read
sha256sum: regtest/zc-testnet-public-alpha-verification.key: Too many levels of symbolic links
regtest/zc-testnet-public-alpha-verification.key: FAILED open or read
sha256sum: testnet3/zc-testnet-public-alpha-verification.key: Too many levels of symbolic links
testnet3/zc-testnet-public-alpha-verification.key: FAILED open or read
sha256sum: WARNING: 4 listed files could not be read
```

Is that a bug in an old fetch-params.sh, or did someone try copying/moving parameters around without realizing they were symlinks?

I moved the directory somewhere else and re-downloaded the parameters.

It still doesn't work because it's trying to read from `/home/zcoin/.zcash-params/zc-testnet-public-alpha-proving.key` not from within `regtest` or `testnet3`. I added more symlinks to fix it and opened #689.

**Comment by defuse** [comment]
Anyway, the testnet is hosted and I redeployed it. This ticket can be closed now.
